### PR TITLE
Story/cord 31

### PR DESCRIPTION
**Pull request by aa-kashk**
add footer to layout

### CODE REVIEW

**Comment by jdamerow** [comment]
Let's talk about this. There are still 30 changed files.

File: cord19-explorer/src/main/webapp/WEB-INF/views/layouts/footer.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+<body>
+	<div th:fragment="footer" th:align="center">
+
+			<div>
+				<h5>
+					Version: <span
+						th:text="${#strings.defaultString(@environment.getProperty('buildNumber'),'')}"></span>
+					<span th:if="${@environment.getProperty('pullReq') != null}">,
+						Pull Request: <span
+						th:text="${#strings.defaultString(@environment.getProperty('pullrequest'),'')}"></span>
```
Comment (jdamerow) [inline cord19-explorer/src/main/webapp/WEB-INF/views/layouts/footer.html]: pullrequest and buildNumber need to be properties in the pom.xml

**Review by jdamerow — CHANGES_REQUESTED** [review]

**Comment by jdamerow** [comment]
Make it so, Jenkins.

File: cord19-explorer/src/main/webapp/WEB-INF/views/layouts/footer.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+<meta charset="UTF-8">
+</head>
+<body>
+	<div th:fragment="footer" th:align="center">
+
+			<div>
+				<h5>
+					Version: <span
+						th:text="${#strings.defaultString(@environment.getProperty('buildNumber'),'')}"></span>
+					<span th:if="${@environment.getProperty('pullReq') != null}">,
```
Comment (jdamerow) [inline cord19-explorer/src/main/webapp/WEB-INF/views/layouts/footer.html]: that should be `pullrequest`

File: cord19-explorer/src/main/webapp/WEB-INF/views/layouts/footer.html (hunk 1 above)
Comment (jdamerow) [reply cord19-explorer/src/main/webapp/WEB-INF/views/layouts/footer.html]: never mind

**Comment by jdamerow** [comment]
I believe you need to add the build number plugin to get buildnumbers. Do you get any locally?

**Comment by aa-kashk** [comment]
The build number plugin is already present in the pom.xml.

**Comment by jdamerow** [comment]
Make it so, Jenkins.

**Comment by jdamerow** [comment]
Make it so, Jenkins.

**Comment by jdamerow** [comment]
Hm, I'm not getting a version number. Do they get generated locally?

**Comment by aa-kashk** [comment]
The config.properties were missing the buildNum and pullReq fields. I added those and the version info isn't null but it still appears empty in my local.

**Comment by jdamerow** [comment]
Seems to work great on staging! 👍
image

**Pull request merged**